packages/api/package.json

```diff
@@ -6,23 +6,20 @@
   "engines": {
     "node": ">=16.19.0"
   },
-  "bin": {
-    "cumulus-api": "./bin/cli.js"
-  },
```

Comment:
This was interesting. At some point this stopped working. Perhaps on the Lerna update several months ago? 

Before this change if you ran one of the scripts that used `cumulus-api` it would say "sh: cumulus-api not found". It appears that anything in the `bin` object isn't installed on bootstrap. 

The fix here is to update the scripts to call "./bin/cli.js" directly but I'm open to pushback.

Reply:
Just to note: we have `bin` commands in other packages but they aren't actually used anywhere. This is the only instance where we're actually using one in our `scripts`